src/screens/Library/components/GameCard/index.css

```diff
@@ -82,89 +80,132 @@
 .gameCard .gameTitle {
   position: absolute;
   width: 100%;
-  bottom: -92px;
+  bottom: min(0.5vh, 46px);
   padding: 0.3vw;
   color: var(--text-secondary);
   display: flex;
   background: var(--gradient-gamecard);
   transition: 200ms;
   font-family: var(--content-font-family);
-  font-size: 14px;
+  font-size: clamp(14px, 1vw, 16px);
   font-weight: 700;
   line-height: 17px;
   color: var(--text-default);
-  height: clamp(80px, 4vw, 85px);
+  height: 35%;
+  max-height: 68px;
   cursor: pointer;
   z-index: 3;
   pointer-events: none;
-  backdrop-filter: blur(var(--blur-light));
+}
+
+.gameCard .gameTitle span {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  max-height: 18px;
+  -webkit-line-clamp: 1;
+  -webkit-box-orient: vertical;
+}
+
+.gameCard:hover .gameTitle {
+  height: 50%;
+  max-height: 90px;
+}
+
+.gameCard:hover .gameTitle span {
+  overflow: visible;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  max-height: fit-content;
+  height: min-content;
+  -webkit-line-clamp: 4;
+  -webkit-box-orient: vertical;
 }
 
 .gameCard > .icons {
-  opacity: 0;
+  opacity: 1;
   position: absolute;
   bottom: 0px;
+  background: var(--icons-background);
+  min-height: 42px;
+  max-height: 56px;
+  height: 3vw;
+  justify-content: space-around;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-areas: 'update settings play';
 }
 
-.gameCard:hover > .icons,
-.gameCard:focus-within > .icons {
-  opacity: 1;
+.gameCard > .icons > .svg-button {
+  justify-self: center;
 }
 
-.gameCard:hover > .gameTitle,
-.gameCard:focus-within > .gameTitle {
```

Comment:
maybe this change can be an option in the accessibility tab? if the reason of the change is accessibility, because the library is kinda cluttered now by default and there's no way back if one wants to use a cleaner layout

Reply:
Do you mean without showing the title all the time?
I don't know, this is how it looks in the current design in Figma. 
On Twitter at least we got good feedback from people, let's release like this on the beta and check what people will say. 👍🏽